.github/workflows/ci.yml

```diff
@@ -193,6 +188,29 @@ jobs:
         python scripts/test_tensor_api.py -n 1
       shell: bash
 
+    - name: Free up disk space
+      run: |
+        sudo rm -rf /usr/share/dotnet
+        sudo rm -rf /opt/ghc
+        sudo rm -rf /opt/hostedtoolcache/CodeQL
```

Comment:
If we are going to free up the disk space, it might be better to free up at the beginning of the job